### DISCUSSION TITLE
Favourite sites

### DISCUSSION THREAD

**Discussion by oscartbeaumont** [plausible/analytics#2113] [Anything Else]
It would be really nice if you could pin a set of your favorite sites on the dashboard. I manage a lot of sites and some of them I rarely need to look at and having a way to push them to the bottom would be nice.

Thanks for all of your great work on this awesome product!

**Comment by metmarkosaric** [comment]
You can now pin sites that you want to feature at the top of your sites page. You can pin up to nine sites. The most recently pinned site will automatically appear at the top of your list, helping you organize your dashboards by how often you use them.

Your pinned sites will also show up in the site switcher list that you can see within each dashboard by clicking on the site name in the top left. You can navigate between the sites in the site switcher list using keyboard shortcuts 1 to 9.
